src/models/searchservice/response/directanswer/RichTextFeaturedSnippetDirectAnswer.ts

```diff
@@ -0,0 +1,14 @@
+import { BuiltInFieldType } from './BuiltInFieldType';
+import { BaseFeaturedSnippetDirectAnswer } from './BaseFeaturedSnippetDirectAnswer';
+import { EnumOrLiteral } from '../../../utils/EnumOrLiteral';
+
+/**
+ * A {@link BaseFeaturedSnippetDirectAnswer} with 'rich_text' field type.
+ * "value" field is omitted for featured snippet direct answer of this field type.
+ *
+ * @public
+ */
+export interface RichTextFeaturedSnippetDirectAnswer extends Omit<BaseFeaturedSnippetDirectAnswer<string>, 'value'> {
```

Comment:
are you sure you don't want to put RichTextFeaturedSnippetDirectAnswer, MultiLineTextFeaturedSnippetDirectAnswer, and BaseFeaturedSnippetDirectAnswer into FeaturedSnippetDirectAnswer.ts ? There's a lot of jumping around when I try to navigate the code, and each file is less than 20 lines and I don't see them getting much bigger in the future

Reply:
It would only add about ~30 lines to FeaturedSnippetDirectAnswer.ts, I feel like that's totally reasonable

Reply:
sure, I can combine them